flashcard_model.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+
+class FlashcardModel
+  attr_reader :file, :answers, :parsed_file
+
+  def initialize(file)
+    @file = file
+    @answers = []
+    @parsed_file = []
```

Comment:
(This would be a fairly big refactor, so I don't necessarily suggest changing this right now.) A more object oriented way to do this, instead of having a nested array, would be to create something like a Card class, that stores the question and answer for a card. And then your FlashcardModel could have an instance variable that holds an array of card objects.